UserInterface/Views/MainWindow.xaml

```diff
@@ -9,8 +9,8 @@
         xmlns:viewModels="clr-namespace:UserInterface.ViewModels"
         xmlns:modalViewModels="clr-namespace:UserInterface.ViewModels.Modals"
         mc:Ignorable="d"
-        Title="Gehoortest" Height="1080" Width="1920" WindowState="Maximized" WindowStyle="None">
-
+        Title="Gehoortest" Height="1080" Width="1920" >
+    
```

Comment:
SPATIE

Reply:
Niet akkoord je mist de maximzed window!

Reply:
oke hij is weg